airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -3150,6 +3151,42 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+  StateDelegatingRetriever:
+    description: (This component is experimental. Use at your own risk.) Orchestrate the retriever's usage based on the state value.
+    type: object
+    required:
+      - type
+      - full_refresh_retriever
+      - incremental_retriever
+    properties:
+      type:
+        type: string
+        enum: [ StateDelegatingRetriever ]
+      full_refresh_no_slice_in_params:
+        description: If set to true, a single slice will be used and its request options will be ignored when sending requests.
+        type: boolean
+        default: false
+      full_refresh_ignore_min_max_datetime:
```

Comment:
In which context do we need this for Stripe?

Reply:
This flag is needed because some endpoints have limitations for incremental reads (e.g., events can only be read up to 30 days in the past).

To avoid restricting the start/end date during a full refresh, we can use this flag.